features/rollershutter_item.feature

```diff
@@ -0,0 +1,71 @@
+Feature: rollershutter_item
+  Rollershutter Items are supported
+
+  Background:
+    Given Clean OpenHAB with latest Ruby Libraries
+    And group "Rollershutters"
+    And items:
+      | type          | name      | label        | group          |
+      | Rollershutter | RollerOne | Roller One   | Rollershutters |
+      | Rollershutter | RollerTwo | Roller Two   | Rollershutters |
+
+  Scenario: Sending command using PercentType work
+    Given item "RollerOne" state is changed to "<initial>"
+    And item "RollerTwo" state is changed to "0"
+    And code in a rules file
+      """
+      java_import Java::OrgOpenhabCoreLibraryTypes::DecimalType
```

Comment:
Should we be testing / supporting this?  Trying to figure out why the user would want to create a decimal type in here and ever use it?

Reply:
I added it because of a question from @jimtng, wouldn't expect it to be used in practice. It might be useful to pass the state from a regular Number Item though, but that conversion is perhaps handled already?

Reply:
I think it should be handled automatically.. its ok to leave the test.

Reply:
All persistence functions return a decimaltype